justfile

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
I'd rather not have a command runner script in the repo (not everyone uses or needs this), is it OK if we remove this from the PR?

Reply:
sure!